src/LinkMe/LinkMe.sln

```diff
@@ -0,0 +1,69 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
possible to use the new SLNX format?
https://blog.ndepend.com/slnx-the-new-net-solution-xml-file-format/

Reply:
still in preview mode, so it's best not to use it.